python/lsst/ts/observatory/control/base_tcs.py

```diff
@@ -63,16 +63,26 @@ class BaseTCS(RemoteGroup, metaclass=abc.ABCMeta):
         used to limit the resources allocated by the class by gathering some
         knowledge about the usage intention. By default allocates all
         resources.
-
+    concurrent_operation : `bool`, optional
+        Performs all tasks sequentially? If `False`, tasks like `enable` and
```

Comment:
Remove `Performs all tasks sequentially?`. I think the rest of the comment works just fine.